<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>391</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,0,0,0">
     <item>
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>顺序播放</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>随机播放</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>列表循环</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>单曲循环</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>导入歌曲</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>30</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>音量</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="verticalSlider">
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>80</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>391</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="music_namelabel">
       <property name="text">
        <string>歌曲名字</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="2">
        <widget class="QToolButton" name="pauseButton">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="image.qrc">
           <normaloff>:/res/img/pause.png</normaloff>:/res/img/pause.png</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QToolButton" name="playButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="image.qrc">
           <normaloff>:/res/img/play.png</normaloff>:/res/img/play.png</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QToolButton" name="leftButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="image.qrc">
           <normaloff>:/res/img/left.png</normaloff>:/res/img/left.png</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QToolButton" name="rightButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="image.qrc">
           <normaloff>:/res/img/right.png</normaloff>:/res/img/right.png</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="musiclistWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>161</width>
      <height>251</height>
     </rect>
    </property>
    <property name="columnCount">
     <number>2</number>
    </property>
    <column/>
    <column/>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>39</x>
      <y>50</y>
      <width>391</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="all_timelabel">
       <property name="text">
        <string>00：00</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>140</y>
      <width>231</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>472</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
